src/main/java/core/basesyntax/strategy/operation/PurchaseOperation.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy.operation;
+
+import core.basesyntax.dao.StorageDao;
```

Comment:
The import statements should be followed by the class declaration without an empty line in between.